src/ActivitiesSDKAdapter.js

```diff
@@ -166,4 +166,44 @@ export default class ActivitiesSDKAdapter extends ActivitiesAdapter {
 
     return activity$;
   }
+
+  /**
+   * A function that checks whether or not an Activity object contains a card attachment.
+   *
+   * @param {Activity} activity  Activity object
+   * @returns {boolean} True if received Activity object contains a card attachment
+   */
+  // eslint-disable-next-line class-methods-use-this
+  hasAdaptiveCard(activity) {
+    return activity.attachments && activity.attachments[0] && activity.attachments[0].contentType === 'application/vnd.microsoft.card.adaptive';
+  }
+
+  /**
+   * A function that returns adaptive card data of an Activity object.
+   *
+   * @param {Activity} activity  Activity object
+   * @returns {object} Adaptive card data object
+   */
+  // eslint-disable-next-line class-methods-use-this
+  getAdaptiveCard(activity) {
+    const hasCard = this.hasAdaptiveCard(activity);
+
+    return hasCard ? activity.attachments[0].content : undefined;
+  }
+
+  /**
+   * A function that attach an adaptive card to an Activity object.
+   *
+   * @param {object} activity  The activity to post
```

Comment:
use `Activity` instead of `object` as a data type https://github.com/webex/sdk-component-adapter/pull/284/files#diff-83bc65c82b97dada43c89addca9ce41a6d31ea6a9107d03feea219a81f8632adR19